shell/imports/sandstorm-db/profile.js

```diff
@@ -230,7 +229,7 @@ SandstormDb.fillInProfileDefaults = function (credential, profile) {
     profile.name = profile.name || "Demo User";
     profile.handle = profile.handle || "demo";
   } else if (services.ldap) {
-    const setting = Settings.findOne({ _id: "ldapNameField" });
+    const setting = globalDb.collections.settings.findOne({ _id: "ldapNameField" });
```

Comment:
strictly speaking, `globalDb` isn't in scope here, so you should import it.  (Yes, that results in a circular dependency, but that's the state of the world today.)

Better would be something like `this.collections.settings.findOne`, given that it's a method being attached to `SandstormDb`, but I think it might have to go on the prototype to get the `this` behavior correct.

In an ideal world, we'd have a real class for `SandstormDb` (and perhaps more precise implementations extending it as `ServerDb` and `ClientDb`) and we'd avoid tossing methods on `SandstormDb` from files other than the one that defines it.

Reply:
Added the import. And yes, we should refactor all of this to be more reasonable at some point.